v2g-liberty-cards/src/car-reservation-calendar-settings-card.ts

```diff
@@ -90,6 +96,28 @@ export class CarReservationCalendarSettingsCard extends LitElement {
       : nothing;
   }
 
+
+  private _renderCaldavConnectionStatus() {
+    const state = this._caldavConnectionStatus.state;
+    const isConnected = state === 'Successfully connected';
+    const hasConnectionError =
+      state === 'Connection error' || state === 'Failed to connect';
+    const error = tp('connection-error');
+    const success = tp('connection-success', {
+      time: elapsedTimeSince(this._caldavConnectionStatus.last_updated),
+    });
+    return isConnected
+      ? html`
+          <div class="success">
```

Comment:
Consider using a `ha-alert alert-type="success"` instead. That keeps you from having to style it yourself.

Reply:
Will do in next PR

Reply:
No, in this case it renders not lake an alert but the box with a green line. This is not a standard HA element so keeping this HTML / style.

Reply:
OK, agree. I think the alert is a bit too 'alarming' but semantically it is the right thing. And I think the design should remain as much default HA as possible (so users can theme if they want). I'll change it.